bcreg-aca/config/services.yml

```diff
@@ -252,10 +254,9 @@ issuers:
 
       - schema: address.registries.ca
         issuer_url: $APPLICATION_URL_ADDRESS
-        description: TODO
         details:
-          label_en: Corporate Address
           url_en: $APPLICATION_URL_ADDRESS
+          credential_title: Address
```

Comment:
I don't think the address credential title is needed. As far as Akiff's told me we aren't really using addresses yet (currently there are no address credentials on prod). When we do start using address credentials we might need to sit down and figure out which attribute to display as the title, but right now the wireframes for the new orgbook don't specify a credential title for addresses